bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableClusterUtilities.java

```diff
@@ -73,6 +70,66 @@ public static BigtableClusterUtilities forAllInstances(String projectId)
   }
 
   /**
+   * @return The instance id associated with the given project, zone and cluster.
+   */
+  public static String lookupInstanceId(String projectId, String clusterId, String zoneId)
+    throws IOException {
+    BigtableClusterUtilities utils;
+    try {
+      utils = BigtableClusterUtilities.forAllInstances(projectId);
```

Comment:
nit:
Does it make more sense if we only allow BigtableClusterUtilities to be instantiated when either an instance ID or a cluster ID is provided? With that, we don't need to pass in any cluster ID or Instance ID in those non-static public methods.

example: no getClusterSize(String clusterId, String zoneId)  and getClusterSize() 
we will only have getClusterSize()